engine/apps/alerts/models/alert_group.py

```diff
@@ -389,9 +389,6 @@ def status(self):
     # https://code.djangoproject.com/ticket/28545
     is_open_for_grouping = models.BooleanField(default=None, null=True, blank=True)
 
-    # TODO: drop this column in an upcoming release
-    is_restricted = models.BooleanField(default=False, null=True)
```

Comment:
Should we probably try the suggested steps?
https://github.com/grafana/oncall/blob/dev/dev/README.md#removing-a-nullable-field-from-a-model

Reply:
good point 👍